src/models/template/schema.js

```diff
@@ -0,0 +1,84 @@
+const _idSchema = {
```

Comment:
```suggestion
const _id = {
```
Nên bỏ đuôi `Schema` đằng sau (tất cả phía dưới nữa), vì ta không còn export subschemas như trước nữa, nên không sợ bị tối nghĩa hay trùng tên gì cả.